### PR TITLE
Fix obs-group styling

### PR DESCRIPTION
**Pull request by samuelmale**
Screenshot 2022-11-07 at 18 53 04

### CODE REVIEW

File: src/components/inputs/_input.scss [hunk 1]
```diff
@@ -193,6 +192,24 @@
   align-items: baseline;
 }
 
+.flex_row {
```
Comment (pirupius) [inline src/components/inputs/_input.scss]: Minor but i think this should be `camelCase` like the others for consistency

File: src/components/inputs/_input.scss [hunk 2]
```diff
@@ -193,6 +192,24 @@
...
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.flex_column {
```
Comment (pirupius) [inline src/components/inputs/_input.scss]: same applies here

File: src/components/repeat/ohri-repeat.component.tsx [hunk 3]
```diff
@@ -115,46 +106,55 @@ export const OHRIRepeat: React.FC<OHRIFormFieldProps> = ({ question, onChange })
   };
   const nodes = questions.map((question, index) => {
     const deleteControl =
-      index !== 0 ? (
-        <Column style={{ paddingTop: '1.2rem', marginLeft: '.5rem' }}>
+      questions.length > 1 ? (
+        <div style={{ paddingTop: '1.2rem', marginLeft: '.5rem' }} className={styles.flex_column}>
           <Button
             renderIcon={() => <TrashCan size={32} />}
```
Comment (pirupius) [inline src/components/repeat/ohri-repeat.component.tsx]: `size={16}`

**Review by pirupius — COMMENTED** [review]
Finally 🥳 Some cosmetic changes to make

File: src/components/inputs/_input.scss (hunk 1 above)
Comment (samuelmale) [reply src/components/inputs/_input.scss]: Nice catch

File: src/components/repeat/ohri-repeat.component.tsx [hunk 4]
```diff
@@ -115,46 +106,57 @@ export const OHRIRepeat: React.FC<OHRIFormFieldProps> = ({ question, onChange })
...
-        <Column style={{ paddingTop: '1.2rem', marginLeft: '.5rem' }}>
-          <Button
-            renderIcon={() => <TrashCan size={32} />}
-            kind="danger--tertiary"
-            onClick={() => removeQuestion(question)}
-            hasIconOnly
-          />
-        </Column>
+      questions.length > 1 ? (
+        <div>
```
Comment (samuelmale) [inline src/components/repeat/ohri-repeat.component.tsx:110]: Added wrapper div to fix:

Screenshot 2022-11-08 at 12 15 29

**Review by pirupius — APPROVED** [review]

**Pull request merged**